avocado/utils/memory.py

```diff
@@ -216,6 +216,25 @@ def numa_nodes():
     return (sorted(nodes))
 
 
+def numa_nodes_with_memory():
+    """
+    Get a list of NUMA nodes present with memory on the system.
+
+    :return: List with nodes which has memory.
+    """
+    nodes = []
+    mem_path = '/sys/devices/system/node/has_normal_memory'
+    if not os.path.exists(mem_path):
+        mem_path = '/sys/devices/system/node/has_memory'
```

Comment:
it'd be nice to return better exception in case this second path doesn't exists as well.